code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -692,7 +692,6 @@ datum
 			overdose_threshold = 200 // Hyperglycaemic shock
 
 			on_mob_life(var/mob/living/M as mob)
-				M.nutrition += 1*REM

```

Comment:
Yup; sugar's benefits are already pretty good, as is for a reagent that's so easy to acquire (and virtually unlimited)--not to mention that sugar, alone isn't very nutritious by itself; It's a good way to separate out "junk" food from nutritional food.